src/object/object_representation.c

```diff
@@ -5809,6 +5822,38 @@ or_header_size (char *ptr)
   return mvcc_header_size_lookup[OR_GET_MVCC_FLAG (ptr)];
 }
 
+/*
+ * or_pack_int_array - write a int array
+ *    return: advanced buffer pointer
+ *    buffer(out): output buffer
+ *    count(in): array length
+ *    int_array(in): int array
+ */
+char *
+or_pack_int_array (char *buffer, int count, int *int_array)
+{
+  int i;
+  char *ptr;
+
+  count = (count >= 0) ? count : 0;
```

Comment:
```suggestion
  if (count < 0)
    {
      count = 0;
    }
```

NIT : 가독성 측면에서 제안드립니다.

Reply:
좋습니다. 반영했습니다.